docs/pages/templates/chat/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Chat'
+---
+
+import { Cards } from 'nextra-theme-docs'
+import { Card } from '../../../components/Card'
+
+# Chat
+
+The Chat template offers a ready-to-use chat application, demonstrating the full capabilities of Liteflow’s wallet-to-wallet messaging. It's designed for easy integration and customization within your existing infrastructure.
+
+<Cards num={2}>
+  <Card noPadding>![Home](/templates/chat/home.png)</Card>
+  <Card noPadding>![Home](/templates/chat/conversation.png)</Card>
+  <Card noPadding>![Home](/templates/chat/embed.png)</Card>
+  <Card noPadding>![Home](/templates/chat/widget.png)</Card>
+</Cards>
+
+### Demo
+
+Explore a live demonstration of the chat template at: https://chat-liteflow.vercel.app/
+
+## Features included
+
+<Cards>
+  <Card title="Direct Messaging">
+    Wallet-to-wallet communication baked into your application.
+  </Card>
+  <Card title="Theme Customization">
+    Tailor the chat interface to complement your brand’s aesthetics.
+  </Card>
+  <Card title="Versatile Widget">
+    An embeddable chat widget that can be placed in any part of your
+    application.
+  </Card>
```

Comment:
Added another section to keep the balance of 6 elements

### Wallet Compatibility
Support for unlimited wallets thanks to [RainbowKit](https://www.rainbowkit.com/)